acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/bulkloading/AzureKeyVaultAcceptanceTest.java

```diff
@@ -162,7 +191,54 @@ void envVarsAreUsedToDefaultAzureParams() {
 
     startSigner(configBuilder.build());
 
-    final Response response = signer.callApiPublicKeys(KeyType.BLS);
-    response.then().statusCode(200).contentType(ContentType.JSON).body("", contains(EXPECTED_KEY));
+    final List<String> publicKeys =
+        multilineKeys.stream()
+            .map(BLSKeyPair::getPublicKey)
+            .map(BLSPublicKey::toHexString)
+            .collect(Collectors.toList());
+    publicKeys.add(EXPECTED_KEY);
```

Comment:
Seems like we've replaced the more common single key scenario with the less common multiKey scenario. 
These are separate tests in AWS, feel like that's a better approach, even if the current implementation is equivalent to a "list of one".